changelogs/unreleased/66728.yml

```diff
@@ -0,0 +1,2 @@
+title: "Stock : refactor stock computeIndicators method orchestration"
```

Comment:
This changelog is more for functionnal than technical, so things as refactor should not be noted in changelog.
However since this work concerns a bug with indicators, the changelog can be "Fixed a bug where indicators were not always computed correctly." Or something like that